src/content/de/updates/2018/01/crux.md

```diff
@@ -10,22 +10,22 @@ description: Announcing the release of a new country dimension in the Chrome Use
 {# wf_featured_image: /web/updates/images/generic/timeline.png #}
 {# wf_featured_snippet: Announcing the release of a new country dimension in the Chrome User Experience Report. #}
 
-# Chrome User Experience Report: Neue Länderdimension {: .page-title }
+# Chrome User Experience-Bericht: Neue {: .page-title }
 
 {% include "web/_shared/contributors/rviscomi.html" %}
 
 <div class="clearfix"></div>
 
-Der [Chrome User Experience-Bericht](/web/tools/chrome-user-experience-report/) (CrUX) ist ein öffentlicher Datensatz realer Benutzerleistungsdaten. Seit wir (1) den Bericht angekündigt haben, war eine der am häufigsten nachgefragten Ergänzungen die Fähigkeit, Unterschiede in der Benutzererfahrung über Standorte hinweg besser zu verstehen. Basierend auf diesem Feedback erweitern wir den bestehenden CrUX-Datensatz, der eine globale Sicht über alle geografischen Regionen hinweg bietet, um auch eine Sammlung von separaten länderspezifischen Datensätzen einzubeziehen!
+Der [Chrome User Experience Report](/web/tools/chrome-user-experience-report/) (CRUX) ist ein öffentlicher Datensatz von realen Benutzerleistungsdaten. Da wir den Bericht [announced](https://blog.chromium.org/2017/10/introducing-chrome-user-experience-report.html) haben, war eine der am häufigsten nachgefragten Ergänzungen die Fähigkeit, Unterschiede in der Benutzererfahrung an verschiedenen Standorten besser zu verstehen. Basierend auf diesem Feedback erweitern wir den bestehenden CrUX-Datensatz, der eine globale Sicht über alle geografischen Regionen hinweg bietet, um auch eine Sammlung von separaten länderspezifischen Datensätzen einzubeziehen!
```

Comment:
NIT: Is there any way we can line break these in the future? It'll make the diff checking a LOT easier. Right now it's hard to tell what actually changed between the lines.